app/src/ui_stream_nbgl.c

```diff
@@ -195,7 +195,15 @@ tz_ui_nav_cb(uint8_t page, nbgl_pageContent_t *content)
         tz_ui_continue();
     }
 
-    if (!s->full && !global.apdu.sign.u.clear.skip_to_sign) {
+    if (s->full) {
+        content->type                        = INFO_LONG_PRESS;
+        content->infoLongPress.icon          = &C_tezos;
+        content->infoLongPress.text          = "Sign";
+        content->infoLongPress.longPressText = "Sign";
+    } else if (page == LAST_PAGE_FOR_REVIEW) {
+        nbgl_useCaseSpinner("Skipping");
```

Comment:
```suggestion
        nbgl_useCaseSpinner("Loading operation");
```
for consistency with blindsigning loading screen?